OpenRA.Mods.Common/EditorBrushes/EditorDefaultBrush.cs

```diff
@@ -51,6 +52,18 @@ public EditorDefaultBrush(EditorViewportControllerWidget editorWidget, WorldRend
 				.ToDictionary(r => r.Info.ResourceType, r => r);
 		}
 
+		long CalculateActorSelectionPriority(EditorActorPreview actor)
```

Comment:
I could do that, but that would:
1) make this depend on #10011.
2) replace visual ordering with in game selection.
Not that these are undesirable.